packages/core/notarization/manager.go

```diff
@@ -159,34 +159,32 @@ func (m *Manager) LoadOutputsWithMetadata(outputsWithMetadatas []*ledger.OutputW
 }
 
 // LoadEpochDiffs updates the state tree from a given snapshot.
-func (m *Manager) LoadEpochDiffs(header *ledger.SnapshotHeader, epochDiffs map[epoch.Index]*ledger.EpochDiff) {
+func (m *Manager) LoadEpochDiffs(ei epoch.Index, epochDiff *ledger.EpochDiff) {
```

Comment:
We can drop the `ei` parameter and rename the method to `LoadEpochDiff` (singular), to highlight that this now loads a single epoch's diff.